### ISSUE TITLE
Hub/NavBar: Keyboard focus rect sometimes shows for mouse focus

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#251]
### Repro Steps
1. In Chrome/Firefox, go to the Hub sample on http://try.buildwinjs.com/#hub
2. Click on a header to give it focus
3. Use the arrows on the keyboard to move focus between headers (This puts things into a bad state. Without this step, the bug won't repro.)
4. Click on a header

**Expected:** The header has focus and doesn't have a keyboard focus rect

**Actual:** The header has focus and the keyboard focus rect is drawn

Repros in Chrome and Firefox.

Doesn't repro in IE.

Other browsers not tested.
